lib/ui/painting.dart

```diff
@@ -5334,18 +5334,18 @@ class _DisplayListCanvas implements Canvas {
   static ByteData _cachedData = _emptyData;
   static Uint32List _emptyInts = _emptyData.buffer.asUint32List();
   static Float32List _emptyFloats = _emptyData.buffer.asFloat32List();
-  static List<Object> _deadObjects = List<Object>.empty(growable: false);
+  static List<NativeFieldWrapperClass2> _deadObjects = List<NativeFieldWrapperClass2>.empty(growable: false);
```

Comment:
I think this could just be `static const List<NativeFieldWrapperClass2> _deadObjects = <NativeFieldWrapperClass2>[];`.

For that matter, you could just inline that into the dispose method below.

Reply:
Originally I was going to use these as a flag for the StateError below to detect when a developer was trying to reuse a Canvas after endRecording, but I instead test for the Recorder link so I don't really need most of these values to be static any more.

I'm not sure of the cost of creating the empty byte data and then creating an int/float buffer from it, though.

Reply:
If it's const it basically is baked into the snapshot and is o(1) to construct at runtime